source/slang/slang-ir-use-uninitialized-out-param.cpp

```diff
@@ -63,6 +63,7 @@ namespace Slang
                         addresses.add(use->getUser());
                         break;
                     case kIROp_Store:
+                    case kIROp_SwizzledStore:
```

Comment:
I wonder if we should either drop swizzledstore altogether, or make a superclass for store and swizzledstore...

Reply:
(this is just musing, not for this PR)